app/models/concerns/search_domain.rb

```diff
@@ -25,4 +28,3 @@ def normalize_domain
 
 end
 
```

Comment:
This empty line should be removed

Reply:
I can remove the space but that was there from before and I was just updating the code.

Reply:
as that file needs a lot of cleanup!! :)

Reply:
I was referring to the empty line at the end of the file, which is the result of the line that was removed (`ActiveRecord::Base.extend SearchDomain`).